addons/web/static/src/webclient/switch_company_menu/switch_company_menu.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
 <t t-name="web.SwitchCompanyMenu" owl="1">
-    <Dropdown class="'o_switch_company_menu d-none d-md-block'" position="'bottom-end'">
+    <Dropdown class="'o_switch_company_menu d-none d-md-block'" position="'bottom-end'" autoOpen="false">
```

Comment:
keep this one autoopen

Reply:
Was for consistency with the behavior of the small icons since it would still open this one on hover while the cursor come from activity. Solved with [brboi](https://github.com/brboi)'s suggestion